src/chapters/terminal/introduction.rst

```diff
@@ -0,0 +1,48 @@
+What is a terminal?
+===================
+
+GUIs and CLIs
+-------------
+
+Most of the time when we use our computers, we do so through a 
+**graphical user interface**, or **GUI** for short. A GUI is a system designed
+with icons and visual representations of the machine's file systems. 
+
+.. figure:: figures/GUI-example.png
+   :alt: Sample graphic user interface.
+
+   A GUI with file icons and columns representing folder structure.
+
+
+Programmers often use another kind of interface, called the **command line**. A 
+**CLI**, or command line interface, uses textual commands, rather than graphic 
+manipulation, to give the computer instructions. 
+
+.. figure:: figures/CLI-example.png
+   :alt: Sample command line interface.
+
+   A CLI with commands navigating the same file paths as the GUI above.
+
+
```

Comment:
This section may be confusing for some readers. Maybe instead of "graphic manipulation", use "drag and drop"?